src/Illuminate/Console/Scheduling/ScheduleRunCommand.php

```diff
@@ -41,12 +42,49 @@ public function __construct(Schedule $schedule)
     }
 
     /**
+     * Get the console command options.
+     *
+     * @return array
+     */
+    public function getOptions()
+    {
+        return [
+            ['loop', 'l', InputOption::VALUE_NONE, 'Don\'t terminate. Run forever'],
+            ['interval', null, InputOption::VALUE_REQUIRED, 'Run every interval seconds. Default: 60', 60],
+        ];
+    }
+
+    /**
      * Execute the console command.
      *
      * @return void
      */
     public function fire()
     {
+        $loop = $this->option('loop');
+        $interval = $this->option('interval');
+        if (! is_int($interval) && $interval <= 0) {
```

Comment:
Is this necessary? If the $interval is <= 0  then it will default to 1 anyway.

And also shouldn't it use || instead of &&?